src/elements/content-uploader/stories/ContentUploader.stories.js

```diff
@@ -1,18 +1,14 @@
-import * as React from 'react';
-import { IntlProvider } from 'react-intl';
+// @flow
 import ContentUploader from '../ContentUploader';
-import notes from './ContentUploader.notes.md';
 
-export const Uploader = () => (
-    <IntlProvider locale="en">
-        <ContentUploader features={global.FEATURES} rootFolderId={global.FOLDER_ID} token={global.TOKEN} />
-    </IntlProvider>
-);
+export const Basic = {};
 
 export default {
-    title: 'Elements|ContentUploader',
+    title: 'Elements/ContentUploader',
     component: ContentUploader,
-    parameters: {
-        notes,
```

Comment:
do we want to keep the notes for this and the other elements? seems like it could provide useful documentation

Reply:
they arent working as intended after the bump, so I removed the prop. we need to use the newer ways of providing documentation to the story